src/sentry/static/sentry/app/components/sidebar/sidebarDropdown/index.tsx

```diff
@@ -133,6 +133,11 @@ const SidebarDropdown = ({api, org, orientation, collapsed, config, user}: Props
                     <SidebarMenuItem to="/settings/account/api/">
                       {t('API keys')}
                     </SidebarMenuItem>
+                    {user.isSuperuser && (
+                      <SidebarMenuItem to="/settings/account/api/mobile-app/">
+                        {t('Mobile app')}
```

Comment:
```suggestion
                        {t('Mobile App')}
```

Reply:
This is for the top left menu where other items are not in title case:
![image](https://user-images.githubusercontent.com/9060071/108523465-3b616480-72ce-11eb-851f-d81f395b911c.png)
Should we still rename it to `Mobile App`?